message_center_compassion/mappings/base_mapping.py

```diff
@@ -136,8 +137,12 @@ def get_connect_data(self, odoo_object, fields_to_submit=None):
                 # Field One2Many
                 if field.endswith('ids'):
                     value = list()
-                    sub_mapping = new_onramp_mapping(
-                        field_mapping[1], self.env)
+                    if field_mapping.__len__() == 2:

```

Comment:
It's for relation one to many for specified the action. 
Example:
        'Pages': ('page_ids', 'correspondence.page', 'specified_Action'),
If specified_action is not give, it take by default 'default'